app/settings/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import { useState } from 'react';
+import { Button, Divider, Spinner } from '@nextui-org/react';
+import { GoDownload } from 'react-icons/go';
+
+export default function Settings() {
+  const [loading, setLoading] = useState(false);
+
+  const handleDownload = () => {
+    setLoading(true);
+    // Simulate download delay
+    setTimeout(() => {
+      setLoading(false);
+      window.location.href = '/api/logs';
+    }, 2000);
+  };
+
+  return (
+    <div className="container mx-auto max-w-6xl py-10 px-6">
+      <h1 className="text-3xl font-bold mb-8">Settings</h1>
+      <Divider className="my-6 w-full" />
+      <div className="space-y-8">
+        <section>
+          <h2 className="text-2xl font-semibold mb-4">Trouble Shooting</h2>
```

Comment:
nit: I normally see this spelled as one word

```suggestion
          <h2 className="text-2xl font-semibold mb-4">Troubleshooting</h2>
```